src/Model/Database.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Model;
+
+
+class Database
+{
+    protected $connection = null;
+
+    public function __construct()
+    {
+        try {
+            $this->connection = new \PDO("mysql:host={$_ENV['DB_HOST']};dbname={$_ENV['DB_NAME']}", getenv('DB_USER'), getenv('DB_PASS'));
+            $this->connection->setAttribute(\PDO::ATTR_EMULATE_PREPARES, false);
+            $this->connection->setAttribute(\PDO::ATTR_DEFAULT_FETCH_MODE, \PDO::FETCH_ASSOC);
+        } catch (\PDOException $e) {
+            print "Error!: " . $e->getMessage() . "\r\n";
+            die();
```

Comment:
die() ist hier gefährlich. Da es das den Request mit status code 200 beendet. Bei einer Webseite vielleicht noch OK aber bei einer REST API gefährlich. Besser wäre es hier gewesen die Exception einfach fliegen zu lassen und diese in der index.php zu fangen.